test/unit_tests/braket/default_simulator/test_state_vector_simulator.py

```diff
@@ -629,7 +629,7 @@ def circuit_noise(ir_type):
             qubit[2] q;
             h q[0];
             cnot q[0], q[1];
-            #pragma braket noise probability(.15) q[0]
```

Comment:
is `probability` still tested elsewhere?

Reply:
probability isn't an actual noise instruction; this was a typo